distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,357 @@
+|etc/org.ops4j.pax.web.cfg
+
+etc/system.properties
+
+etc/ws-security/server/encryption.properties
+
+etc/ws-security/server/signature.properties
+
+etc/ws-security/issuer/encryption.properties
+
+etc/ws-security/issuer/signature.properties
+
+etc/system.properties
+|Used to secure (SSL) all of the endpoints for ${branding}, to perform outgoing SSL requests, and sign STS SAML assertions.
+This also includes the ${admin-console} and any other web service that is hosted by ${branding}.
+
+|===
+
+[NOTE]
+====
+The following information was sourced from https://www.racf.bnl.gov/terapaths/software/the-terapaths-api/example-java-client/java-client/setting-up-keystores-with-jetty-and-keytool.
+====
+
+====== Creating a Client Keystore
+
+[WARNING]
+====
+This walk-through details how to use a PKCS12 store. This is the most popular format used when exporting from a web browser.
+====
+
+. Obtain a personal ECA cert (client certificate)
+.. To do this, open *Internet Explorer -> Tools -> Options*.
+.. Select the *Content* tab.
+.. Click *Certificates*
+.. Select the *Personal* tab.
+.. Select the _certificate_ needed to export. There should be the one without a "Friendly Name" and it is not the "Encryption Cert").
+.. Click *Export*.
+.. Follow *Certificate Export Wizard*.
+.. When a prompt requests to export the private key, select *Yes*.
+. Add a cert to a new java keystore, replacing cert with the name of the PKCS12 Keystore needed to convert, and replace MY_KEYSTORE.jks with the desired name of the Java Keystore:
+
+[source]
+----
+keytool -importkeystore -srckeystore [MY_FILE.p12] -srcstoretype pkcs12
+ -srcalias [ALIAS_SRC] -destkeystore [MY_KEYSTORE.jks]
+ -deststoretype jks -deststorepass [PASSWORD_JKS] -destalias [ALIAS_DEST]
+----
+. The command prompts for two passwords:
+.. Input keystore passphrase is the passphrase that is used to protect cert.p12
+.. Output keystore passphrase is the passphrase that is set for the new java keystore serverKeystore.jks
+. It is recommended that the private key password be the same as the keystore password due to limitations in java.
+.. Run the following command to determine the alias name of the added current entry. It is listed after "Alias Name:"
+
+[source]
+----
+keytool -list -v -keystore serverKeystore.jks
+----
+.. Clone the existing key using the java keytool executable, filling in <CurrentAlias>, <NewAlias>, serverKeystore.jks, and password with the correct names.
+
+[source]
+----
+keytool -keyclone -alias "<CurrentAlias>" -dest "<NewAlias>" -keystore serverKeystore.jks -storepass password
+----
+.. When prompted for a password, use the same password used when the keystore was created.
+.. Delete the original alias
+
+[source]
+----
+keytool -delete -alias "<CurrentAlias>" -keystore serverKeystore.jks -storepass password
+----
+
+[NOTE]
+====
+After the keystore is successfully created, delete the jetty files used to perform the import.
+====
+
+====== Creating a Truststore
```

Comment:
We can now manage the trust store through the admin UI, so delete this entire section (lines 105-131).